mobile/src/main/java/org/hawkular/client/android/backend/model/Alert.java

```diff
@@ -40,10 +40,10 @@
     private String status;
 
     @SerializedName("ctime")
-    private long timestamp;
+    private long ctime;
```

Comment:
@pilhuhn Unfortunately I cannot change the naming convention because the JSON I get has "ctime" as a parameter. Retrofit will not parse it if we give it any other name, that is the reason I changed 'timestamp' -> 'ctime'